pubspec.lock

```diff
@@ -1463,5 +1503,5 @@ packages:
     source: hosted
     version: "3.1.2"
 sdks:
-  dart: ">=3.0.0 <4.0.0"
+  dart: ">=3.1.0-185.0.dev <4.0.0"
```

Comment:
was that changed manually?

Reply:
No, not intentionally.
To resolve this, I have regenerarted it in https://github.com/TUM-Dev/Campus-Flutter/pull/82/commits/35590ba516014b26dab916c60d99105e44e60bdc via this button
![image](https://github.com/TUM-Dev/Campus-Flutter/assets/26258709/990e641a-0e59-47b2-8c39-9fb773a03bd1)